src/include/net_connect.h

```diff
@@ -101,8 +101,8 @@ typedef unsigned long pbs_net_t;        /* for holding host addresses */
 #define IS_SHUTDOWN		8
 #define IS_IDLE			9
 #define IS_ACKOBIT		10
-#define IS_GSS_HANDSHAKE	11
-#define IS_CLUSTER_KEY		12
+#define IS_GSS_HANDSHAKE	11	/* Depricated */
+#define IS_CLUSTER_KEY		12	/* Depricated */
```

Comment:
Deprecated

Reply:
Fixed and also fixed spelling of comment on IS_HPCBP_ATTRIBUTES line, which misled me.